### PR TITLE
update Travis to build and push v1.1.0 images

### PR DESCRIPTION
**Pull request by Madhu-1**

### CODE REVIEW

**Comment by Madhu-1** [comment]
@humblec this  should push v1.1.0 image to the quay

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ branches:
   only:
     - csi-v0.3
     - master
+    - release-v1.1.0
```
Comment (humblec) [inline .travis.yml]: @Madhu-1 do we need to remove csi-v0.3 ?

Comment (Madhu-1) [reply .travis.yml]: removed

**Comment by humblec** [comment]
@Madhu-1 the tests are failing .. not sure why.

**Comment by Madhu-1** [comment]
@humblec CI should pass now PTAL

**Comment by humblec** [comment]
@Madhu-1 Cool. Merging .

**Pull request merged**